cli.js

```diff
@@ -51,6 +51,13 @@ var argv = require('yargs')
     default: defaults.silent,
     global: true
   })
+  .option('tag-prefix', {
+    alias: 't',
+    describe: 'Set a custom prefix for the git tag to be created',
+    type: 'boolean',
```

Comment:
Thanks for feedback! I didn't notice the test failed, sorry about that. Corrected now, but I see merge conflict. I'll see if I can rebase.